### DISCUSSION TITLE
Parents evening booking

### DISCUSSION THREAD

**Discussion by cleder** [Jesulayomy/SCHub#30] [Ideas]
Organizing slots for parents evenings can be a major hassle for teachers. 
Parents can login into the system and book a slot
Booking confirmation and reminders are sent out via email
I can flesh this out further if you think it is needed.

**Comment by Jesulayomy** [comment]
Hello again, I haven't even thought of this! Thanks so much. I do need more help though. I haven't setup email system yet, and I still need some help understanding how larger systems are put together.
I will appreciate any further explanations you can give. Also do you think I should make this a separate project/repository, or just another app. I've also been thinking about porting to Django api, because it seems adding features would much simpler on there.

**Comment by cleder** [comment]
Sorry, I am a bit swamped myself, (I am also a project maintainer so hacktoberfest is busy), I can come up with more details (I implemented this at a prior job) 

Django is a good framework for this, it may attract more contributors

**Reply by Jesulayomy** [reply]
If the project you're referring to is open-source, can I have a link to it? I can work on a Django move I believe it would be best and attractive.

**Reply by cleder** [reply]
The hacktoberfest projects I was talking about are pure python, low level libraries for geospatial data. [FastKML](https://github.com/cleder/fastkml) and [PyGeoIf](https://github.com/cleder/pygeoif).

**Reply by cleder** [reply]
Around schools I have an old (8 years) [open source project with django](https://github.com/cleder/os-opendata-edubase) that has not been maintained for quite some time [Video](https://www.linkedin.com/in/christianledermann/overlay/1475086355431/single-media-viewer/?profileId=ACoAAAeH9o8BpyNqJgyfASGwVt4dJrb76fUfUvs)

**Reply by Jesulayomy** [reply]
Taking a look at it too. I'll try to make up a plan to cover this and some other ideas, this project would need a major overhaul soon 🖥️
